demo-app/src/app/app.component.ts

```diff
@@ -4,7 +4,7 @@ import { MatSidenav } from "@angular/material/sidenav";
 import { BreakpointObserver, BreakpointState } from "@angular/cdk/layout";
 import { of, Subscription } from "rxjs";
 
-const MEDIA_MATCH: string = "(max-width: 600px)";
+const MEDIA_MATCH = "(max-width: 600px)";
```

Comment:
@christophercr 
Just for my personal culture, why do we need to remove the type for the constante ?
thanks

Reply:
Ah, because TypeScript is able to _infer_ the correct type of the variable when it is immediately initialized with a certain value, for example in this case, TypeScript can _infer_ that the type of this variable is **string** 😉 

This helps to keep code concise and less verbose. This is the TSLint rule helping us to detect those cases: https://palantir.github.io/tslint/rules/no-inferrable-types/

Reply:
great, good to know. Thanks for the info.